src/components/TabBar.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+
+const TabBar = () => {
+  const [activeTab, setActiveTab] = useState("home");
+
+  const tabItems = [
+    { key: "home", label: "Home" },
+    { key: "setting", label: "Setting" },
+  ];
+
+  return (
+    <div className="mt-5 flex bg-white px-4 text-black">
+      {tabItems.map((tabItem) => (
```

Comment:
```suggestion
      {tabItems.map(({key, label) => (
```
객체 구조 분해 할당을 사용하여 리팩토링하면 좋을 것 같아요!

Reply:
리뷰 반영했습니다!